src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -235,5 +248,11 @@ PyMODINIT_FUNC PyInit_pythonmonkey(void)
     Py_DECREF(pyModule);
     return NULL;
   }
+
+  SpiderMonkeyError = PyErr_NewException("pythonmonkey.SpiderMonkeyError", NULL, NULL);
+  if (PyModule_AddObject(pyModule, "SpiderMonkeyError", SpiderMonkeyError)) {
```

Comment:
Not working, 
`PyErr_SetString(SpiderMonkeyError, "..."); return NULL;` leaves a `SystemError: initialization of pythonmonkey failed without raising an exception`

Reply:
Move to the beginning of `PyInit_pythonmonkey` function, before the first `SpiderMonkeyError` usage